test/automation/src/sql/notebook.ts

```diff
@@ -232,19 +230,19 @@ export class TextCellToolbar {
 	}
 
 	public async codifySelectedText(): Promise<void> {
-		await this.clickToolbarButton('Code');
+		await this.clickToolbarButton('Insert code');
 	}
 
 	public async insertLink(): Promise<void> {
 		throw new Error('Method not implemented.');
 	}
 
 	public async insertList(): Promise<void> {
-		await this.clickToolbarButton('List');
+		await this.clickToolbarButton('Insert list');
 	}
 
 	public async insertOrderedList(): Promise<void> {
-		await this.clickToolbarButton('Ordered list');
+		await this.clickToolbarButton('Insert ordered list');
```

Comment:
Were these just broken or something?

Reply:
Yeah, they weren't being used before. I only had the bolded one running in the general tests.